.github/workflows/test.yaml

```diff
@@ -47,34 +49,36 @@ jobs:
-#    steps:
-#      - uses: actions/checkout@v2
-#      - name: Set up Python 3.7
-#        uses: actions/setup-python@v2
-#        with:
-#          python-version: 3.7
-#      - name: Install dependencies
-#        run: |
-#          python -m pip install --upgrade pip
-#          python -m pip install --upgrade setuptools
-#          python -m pip install codecov
-#          python -m pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-2.0.0.dev0-cp37-cp37m-manylinux2014_x86_64.whl
-#          if [ -f requirements-test.txt ]; then python -m pip install -r requirements-test.txt; fi
-#          HOROVOD_WITH_GLOO=1 HOROVOD_WITHOUT_MPI=1 HOROVOD_WITHOUT_MXNET=1 pip install git+https://github.com/horovod/horovod.git
-#      - name: Install package
-#        run: |
-#          python -m pip install -e .
-#      - name: Run Examples
-#        run: |
-#          pushd examples/
-#          echo "running ray_ddp_example.py" && python ray_ddp_example.py --smoke-test
-#          echo "running ray_ddp_example.py with Tune" && python ray_ddp_example.py --smoke-test --tune
-#          echo "running ray_ddp_tune.py" && python ray_ddp_tune.py --smoke-test
-#          echo "running ray_horovod_example.py" && python ray_horovod_example.py --smoke-test
-#          echo "running ray_horovod_example.py with Tune" && python ray_horovod_example.py --smoke-test --tune
+  test_linux_ray_master_examples:
+    runs-on: ubuntu-latest
+    timeout-minutes: 12
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade setuptools
+          python -m pip install codecov
+          python -m pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-2.0.0.dev0-cp37-cp37m-manylinux2014_x86_64.whl
+          if [ -f requirements-test.txt ]; then python -m pip install -r requirements-test.txt; fi
+          HOROVOD_WITH_GLOO=1 HOROVOD_WITHOUT_MPI=1 HOROVOD_WITHOUT_MXNET=1 pip install git+https://github.com/horovod/horovod.git
+          # Install PTL master
+          pip install -U https://github.com/PyTorchLightning/pytorch-lightning/archive/master.zip
+      - name: Install package
+        run: |
+          python -m pip install -e .
+      - name: Run Examples
+        run: |
+          pushd examples/
+          # echo "running ray_ddp_example.py" && python ray_ddp_example.py --smoke-test
+          # echo "running ray_ddp_example.py with Tune" && python ray_ddp_example.py --smoke-test --tune
+          # echo "running ray_ddp_tune.py" && python ray_ddp_tune.py --smoke-test
+          # echo "running ray_horovod_example.py" && python ray_horovod_example.py --smoke-test
+          # echo "running ray_horovod_example.py with Tune" && python ray_horovod_example.py --smoke-test --tune
```

Comment:
do we plan to not run any of these?

Reply:
These all use the MNIST dataset from torchvision which is failing right now due to this error https://discuss.pytorch.org/t/mnist-server-down/114433. After the next torchvision release we can re-enable these tests (and the ones on the Ray repo).